cli/azd/pkg/project/framework_service.go

```diff
@@ -26,17 +27,14 @@ const (
 )
 
 func parseServiceLanguage(kind ServiceLanguageKind) (ServiceLanguageKind, error) {
-	if string(kind) == "" {
-		return ServiceLanguageKind(""), fmt.Errorf("language property must not be empty")
-	}
-
```

Comment:
What happen when language is empty? why is this required on this PR for Helm?

Reply:
When we talk about using containers we lose the pre-requisite of their having to be any code in the project.  I could use `azd` to provision infra and deploy a helm chart to an AKS cluster without having any of my own code in the project.

In this case there isn't any language and this sets the president.

Moving forward in ACA - users may also want to bring their own container image and again - deploy a prebuilt/public image to ACA without having any local project code.